src/proteus/dswx_hls.py

```diff
@@ -4217,7 +4391,7 @@ class in the browse image. False to display PSW-Agg.
         landcover_mask = create_landcover_mask(
```

Comment:
I think there's an issue with how the no data value is being set (and colored) for the LAND layer.

Running `gdalinfo` on the LAND.tif layer, the normal `NoData Value=...` field is missing, and the colortable entry assigned for value 255 is `255: 255,255,255,255`. (The other layers have this set as `255: 0,0,0,0`).

I'm not sure what's happening; could you please look into this? The Product Spec document says that there should be a fill value for this layer.